src/core/api/chart_label.go

```diff
@@ -62,8 +57,23 @@ func (cla *ChartLabelAPI) Prepare() {
 	cla.chartFullName = fmt.Sprintf("%s/%s:%s", project, chartName, version)
 }
 
+func (cla *ChartLabelAPI) hasPermission(action rbac.Action) bool {
+	resource := rbac.NewProjectNamespace(cla.project.ProjectID).Resource(rbac.ResourceHelmChartVersionLabel)
+
+	if !cla.SecurityCtx.Can(action, resource) {
+		cla.HandleForbidden(cla.SecurityCtx.GetUsername())
```

Comment:
Do we need to handle 401?

Reply:
401 is handled in the `Prepare` of the ChartLabelAPI